src/components/MovieDetails.js

```diff
@@ -37,8 +38,9 @@ class MovieDetails extends Component {
           <p><b>Type: </b>{details.type}</p>
           <b>Genres: </b>
           <ul>
-              {details.genres}
-            </ul>
+              {details.genres.map(gener => { return <li key={gener}>{gener} </li>;}
```

Comment:
we can make the arrow function more simple

<ul>
            {details.genres.map(gener => (
              <li key={gener}>{gener} </li>
            ))}
          </ul>